composer.json

```diff
@@ -59,6 +59,10 @@
         ],
         "cs:php:fix": [
             "php-cs-fixer fix"
+        ],
+        "pre-commit": [
+            "@static:analyze",
+            "@test"
```

Comment:
I'm afraid this won't work without extra dependency?

Reply:
What do you mean with this one? I added one 'script' that calls the ones that are already defined to make it 'easier' to just run one script before committing, instead of multiple :-)

Reply:
"pre-commit" won't be automatically executed pre commit as a git hook. 
Personally I would change this name to “build” since it's easier to type than “pre-commit”